src/index.ts

```diff
@@ -27,18 +27,29 @@ function defaultMdjsConf(
   }
 }
 
-export async function compile({
+async function compile({
```

Comment:
not a reexport, so was good as it was... but yeah, I remember you like to group all exports like below in this file